Stripe/STPPaymentMethodListDeserializer.h

```diff
@@ -0,0 +1,25 @@
+//
+//  STPPaymentMethodListDeserializer.h
+//  StripeiOS
+//
+//  Created by Yuki Tokuhiro on 5/16/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "STPAPIResponseDecodable.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class STPPaymentMethod;
+
+/**
+ Deserializes the response returned from https://stripe.com/docs/api/payment_methods/list
+ */
+@interface STPPaymentMethodListDeserializer : NSObject <STPAPIResponseDecodable>
```

Comment:
hmm, kinda surprised we don't hit any other list endpoints. I suspect making this a template class would be useful, but feel free to file a followup for that since we don't have a use case now :P

Reply:
ooh good call.  https://jira.corp.stripe.com/browse/IOS-1227